ddl/column.go

```diff
@@ -1292,7 +1286,7 @@ func (w *worker) updateColumnAndIndexes(t table.Table, oldCol, col *model.Column
 		}
 
 		// Update the element in the reorgCtx to keep the atomic access for daemon-worker.
-		w.reorgCtx.setCurrentElement(reorgInfo.elements[i+1])
+		reorgInfo.d.getReorgCtx(reorgInfo.Job).setCurrentElement(reorgInfo.elements[i+1])
```

Comment:
How about w.ddlCtx.getReorgCtx?

Reply:
well, there are ddlCtx in worker, actually I think we can remove the ctx in `reorgInfo`